deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -35,7 +35,7 @@
          (string/join))))
 
 (defn- get-page-reference
-  [block supported-formats]
```

Comment:
All of the supported-formats references as they only existed for this one branch which is no longer needed